test-app/__tests__/webpack.config.babel.js

```diff
@@ -0,0 +1,29 @@
+import test from 'ava'
```

Comment:
This test file shouldn't be called `webpack.config.babel.js`. How about `webpack-config.test.js`?

Reply:
Yep good for me, is indeed better.